bundle-size-chart/package.json

```diff
@@ -17,6 +17,7 @@
     "dev": "nodemon",
     "deploy": "gcloud --project amp-bundle-size-chart app deploy",
     "deploy-cron": "gcloud --project amp-bundle-size-chart app deploy cron.yaml",
+    "deploy-tag": "git tag 'deploy-bundle-size-chart-'`date '+%Y%m%d%H%M%S'`",
```

Comment:
This is totally optional, but if added in each app it'll make it easier to generate the lengthy tag names.